Modules/_sqlite/module.c

```diff
@@ -111,12 +111,15 @@ _sqlite3.enable_shared_cache as pysqlite_enable_shared_cache
 
 Enable or disable shared cache mode for the calling thread.
 
-Experimental/Non-standard.
+This method is deprecated and will be removed in Python 3.12.
```

Comment:
This carries on changes from https://github.com/python/cpython/commit/ddb5e11683c8db9d2095f2f9406701443c4dc9b3